### PR TITLE
Added deps.json to pull in TIA.exe from blob storage

### PR DESCRIPTION
**Pull request by acesiddhu**

### CODE REVIEW

File: Tasks/VsTest/dependencies.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "archivePackages": [
+        {
+            "archiveName": "TestSelector.zip",
+            "url": "https://testselector.blob.core.windows.net/testselector/3110153/TestSelector.zip",
```
Comment (allendm-msft) [inline Tasks/VsTest/dependencies.json]: can we use a short link for the url?

**Comment by allendm-msft** [comment]
add the 3rd party notices
looks fine

**Comment by prawalagarwal** [comment]
looks fine

**Comment by acesiddhu** [comment]
as discussed offline not adding the 3rd party file

**Pull request merged**